go/parquet/pqarrow/schema.go

```diff
@@ -125,7 +125,7 @@ func isDictionaryReadSupported(dt arrow.DataType) bool {
 }
 
 func arrowTimestampToLogical(typ *arrow.TimestampType, unit arrow.TimeUnit) schema.LogicalType {
-	utc := typ.TimeZone == "" || typ.TimeZone == "UTC"
+	isAdjustedToUTC := typ.TimeZone != ""
```

Comment:
shouldn't we also be checking if the timezone is already explicitly "UTC" / "utc" ?

Reply:
That scenario should have the same logic (`isAdjustedToUTC=true`) as any non-empty timezone, so it should already be covered by this,